tools/packaging/common/istio-start.sh

```diff
@@ -103,19 +100,19 @@ ISTIO_AGENT_FLAGS=${ISTIO_AGENT_FLAGS:-}
 # Split ISTIO_AGENT_FLAGS by spaces.
 IFS=' ' read -r -a ISTIO_AGENT_FLAGS_ARRAY <<< "$ISTIO_AGENT_FLAGS"
 
+export MESH_CONFIG=${MESH_CONFIG:-"
+defaultConfig:
+  serviceCluster: $SVC
+  controlPlaneAuthPolicy: ${CONTROL_PLANE_AUTH_POLICY}
```

Comment:
why cant we make this mandatory? with istiod,

Reply:
Blocked by https://github.com/istio/istio/pull/21658 I believe. I can followup